mongo/dataset_store.go

```diff
@@ -172,25 +145,19 @@ func (m *Mongo) GetEditions(ctx context.Context, id, state string, offset, limit
 	if err != nil {
 		log.Event(ctx, "error counting items", log.ERROR, log.Error(err))
 		if err == mgo.ErrNotFound {
-			return &models.EditionUpdateResults{
-				Items:      []*models.EditionUpdate{},
-				Count:      0,
-				TotalCount: 0,
-				Offset:     offset,
-				Limit:      limit,
-			}, nil
+			return []*models.EditionUpdate{}, 0, nil
 		}
-		return nil, err
+		return nil, 0, err
 	}
 
 	if totalCount < 1 {
-		return nil, errs.ErrEditionNotFound
+		return nil, 0, errs.ErrEditionNotFound
 	}
 
 	var results []*models.EditionUpdate
 
 	if limit > 0 {
-		iter := q.Sort().Skip(offset).Limit(limit).Iter()
+		iter := q.Skip(offset).Limit(limit).Iter()
```

Comment:
There is another case of the sort being removed here. If they are going back in then we should be consistent and add it here as well.

Reply:
ah yeah, have added this back in

Reply:
Thank you for spotting this @CarlHembrough